lib/api/actions/functions.php

```diff
@@ -536,76 +540,95 @@ function _beans_add_anonymous_action( $hook, $callback, $priority = 10, $args =
 
 }
 
+// @codingStandardsIgnoreEnd
```

Comment:
This should not be added, if code changes, they need to be compliant.

Reply:
@ThierryA See the note above.